web/themes/custom/pineapple/sass/layouts/nodes/_author.scss

```diff
@@ -1,52 +1,108 @@
-.author-container {
-  margin-bottom: $unit *4;
-  padding: $unit $unit*3;
-  background-color: $light-gray;
+.author-body {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
 
   @include breakpoint($lap) {
-    display: flex;
     flex-direction: row;
+    align-items: flex-start;
+    margin-top: $unit *4;
   }
 
-  .author-info-container {
+  .sidebar-container {
     display: flex;
     flex-direction: column;
-    align-items: center;
+    height: fit-content;
+    width: 100%;
 
     @include breakpoint($lap) {
-      flex-direction: row;
+      flex: 0 0 30%;
+      max-width: 30%;
     }
 
-    .profile-image {
-      width: 75%;
-      max-width: 290px;
-      margin-top: -30%;
-      margin-right: 0;
-      z-index: 1;
-
-      @include breakpoint($lap) {
-        width: 200%;
-        margin-top: -15%;
-        margin-right: 32px;
-      }
+    .author-container {
+      background-color: $light-gray;
 
-      @include breakpoint($desk) {
+      .cover-image-container {
         width: 100%;
+
+        img {
+          width: 100%;
+          object-fit: cover;
+          height: 100%;
+        }
       }
 
-      img {
-        border-radius: 50%;
-        width: 100%;
-        object-fit: cover;
+      .author-name {
+        padding: 0 $unit*3;
+
+        .profile-image {
+          width: 30%;
+          max-width: 290px;
+          margin-top: -15%;
+          z-index: 1;
+
+          @include breakpoint($lap) {
+            width: 100%;
+            margin-top: -30%;
+          }
+
+          @include breakpoint($desk) {
+            width: 50%;
+          }
+
+          img {
+            border-radius: 50%;
+            width: 100%;
+            object-fit: cover;
+          }
+        }
+      }
+
+      .author-info {
+        display: flex;
+        justify-content: space-between;
+        margin: $unit $unit*3;
+
+        .member-since,
+        .roles {
+          p {
+            margin: 0;
+          }
+        }
```

Comment:
The paragraphs you're targeting are the only ones inside author information so it's not necessary to include the member since or roles classes here, so take the p tag out of nesting.

Reply:
done